hugegraph-dist/src/assembly/travis/install-hbase.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
```

Comment:
define a var like BASE_DIR for `dirname $0`

Reply:
done